server/server.go

```diff
@@ -215,7 +217,17 @@ func (s *Server) Tick() {
 
 func (s *Server) ping(w http.ResponseWriter, r *http.Request) {
 	uptime := fmt.Sprintf("%v", time.Since(s.StartTime))
-	err := json.NewEncoder(w).Encode(pingResponse{Uptime: uptime})
+	err := json.NewEncoder(w).Encode(pingResponse{Uptime: uptime, Healthy: "OK!"})
+	if err != nil {
+		mlog.Error("Failed to write ping", mlog.Err(err))
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+}
+
+func (s *Server) healthz(w http.ResponseWriter, r *http.Request) {
```

Comment:
Wondering if there a reason we are duplicating the code?

Reply:
Do you mean to use the same controller function, no? 
```
"/healthz", s.ping
```

Reply:
Yeah, same controller function, or same endpoint. Either works.

Reply:
I started creating `/healthz` endpoint and controller function and then noticed that `/` actually does nothing and only shows uptime. I didn't want to use `/` as endpoint or its underlying controller function for health checking (and liveness and readiness probes) because 1) `/healthz` is a very known endpoint in this space 2) at later time `/` might get changed.

Although I should have created `healthResponse` struct and not reuse `pingResponse` (which I'm gonna change).

Reply:
IMO, _if_ we change the `/` endpoint, then we can easily create a separate handler at that point. Pre-emptively duplicating code thinking of a future change which hasn't happened yet seems redundant.

Reply:
Fair enough, I'll point `/healthz` endpoint to existing ping controller.